src/main/java/org/eclipse/cargotracker/interfaces/booking/facade/internal/DefaultBookingServiceFacade.java

```diff
@@ -92,6 +101,28 @@ public void changeDeadline(String trackingId, Date arrivalDeadline) {
 		return routes;
 	}
 
+	@Override
+	public List<TrackingId> listAllTrackingIds() {
+		return cargoRepository.getAllTrackingIds();
```

Comment:
We should remove getAllTrackingIds from the repository and use the generic find all. The tracking IDs can be extracted from the domain objects in the facade.